From 0b0e5ee325f2890dc54072f6b24c401978f94405 Mon Sep 17 00:00:00 2001
From: Tong Li <litong01@users.noreply.github.com>
Date: Thu, 10 Mar 2022 10:05:52 -0500
Subject: Fixed metallb master branch issue (#37849)

latest metallb master branch is broken. Switch to use the latest
working version v0.12

Signed-off-by: Tong Li <litong01@us.ibm.com>
---
 samples/kind-lb/setupkind.sh | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/samples/kind-lb/setupkind.sh b/samples/kind-lb/setupkind.sh
index 235091110d..ccecdf9647 100755
--- a/samples/kind-lb/setupkind.sh
+++ b/samples/kind-lb/setupkind.sh
@@ -57,10 +57,9 @@ fi
 # Setup cluster context
 kubectl cluster-info --context "kind-${CLUSTERNAME}"
 
-# Setup metallb
-kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/master/manifests/namespace.yaml
-kubectl create secret generic -n metallb-system memberlist --from-literal=secretkey="$RANDOM-$RANDOM-$RANDOM-$RANDOM-$RANDOM"
-kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/master/manifests/metallb.yaml
+# Setup metallb using v0.12.
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12/manifests/namespace.yaml
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12/manifests/metallb.yaml
 
 # The following makes sure that the kube configuration for the cluster is not
 # using the loopback ip as part of the api server endpoint. Without this,
-- 
2.35.3

